src/analyze/analyze.c

```diff
@@ -1312,6 +1314,29 @@ static int dump(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
+static int cat_config(int argc, char *argv[], void *userdata) {
+        char **arg;
+        int r;
+
+        (void) pager_open(arg_no_pager, false);
+
+        STRV_FOREACH(arg, argv + 1) {
+                if (arg != argv + 1)
+                        puts("");
+
+                if (path_is_absolute(*arg)) {
+                        log_error("Arguments must be config file names (relative to /etc/");
+                        return -EINVAL;
+                }
```

Comment:
as mentioned on telegram, I think it might be nice to handle absolute paths here too, i.e. search for them in prefix list, and only refuse them when they aren't in there. Then, strip the specified prefix, and print a clear message saying that the prefix is ignored, and we match by suffix only...

But I guess that'd be an improvement that can happen latter